src/main/java/org/elasticsearch/common/joda/Joda.java

```diff
@@ -133,6 +134,10 @@ public static FormatDateTimeFormatter forPattern(String input, Locale locale) {
             formatter = ISODateTimeFormat.yearMonth();
         } else if ("yearMonthDay".equals(input) || "year_month_day".equals(input)) {
             formatter = ISODateTimeFormat.yearMonthDay();
+        } else if ("epoch_second".equals(input)) {
```

Comment:
We already do not support camelCase for all settings, and I don't think there is any consistency even within the same query/field type/whatever.